play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/config/StandardPhaseBanner.scala

```diff
@@ -51,7 +51,7 @@ trait StandardPhaseBanner {
 }
 
 class StandardBetaBanner extends StandardPhaseBanner {
-  def apply(url: String)(implicit messages: Messages): PhaseBanner = apply(phase = "beta", url = url)
+  def apply(url: String)(implicit messages: Messages): PhaseBanner = apply(phase = "BETA", url = url)
```

Comment:
should these be Beta and Alpha rather than BETA and ALPHA - the [examples on govuk](https://design-system.service.gov.uk/components/phase-banner/) are just capitalized but not fully uppercased

Reply:
I think we were just trying to maintain parity with what's there right now - but it feels like the research that led to this change found that Sentence case was more accessible, so maybe we should change them?  Is it maybe a question for DR?

Reply:
Let's use sentence case rather than capitalised and put it in the release notes.

Reply:
I asked in int-plat-design and Tim has said to use Alpha / Beta 🙌